### PR TITLE
Make osn-tests easier to run locally and in appveyor

### PR DESCRIPTION
**Pull request by pdamasceno**
There is one change to the path of obs-studio-node in imports of all tests that had to be made in order to make it all work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,14 +18,27 @@
     "clean": "git clean -fxd",
     "build:bindings": "cmake-js configure -a x64 --CDCMAKE_PREFIX_INSTALL=distribute && cmake --build build --target install --config Release",
     "build:javascript": "tsc -p js",
-    "build:docs": "typedoc --out docs"
+    "build:docs": "typedoc --out docs",
+    "local:config": "yarn install && git submodule update --init --recursive --force && cmake -Bbuild -H. -G\"Visual Studio 15 2017\" -A x64 -DCMAKE_INSTALL_PREFIX=\"./obs-studio-node\"",
+    "local:build": "cmake --build build --target install",
```
Comment (EddyGharbi) [inline package.json]: --config RelWithDebInfo/Debug is missing here.

Comment (pdamasceno) [reply package.json]: It uses Debug by default if --config is not used. Still want me to add it there?

Comment (EddyGharbi) [reply package.json]: I would make it explicit, doesn't really hurt and prevent some potential mistakes.

Comment (pdamasceno) [reply package.json]: Made it explicit in the latest commit

**Pull request merged**